docs/description.md

```diff
@@ -377,6 +376,88 @@ computations.
 
 🚧 Under construction 🚧
 
+### Use AWS Bedrock Models
```

Comment:
This description.md is not used anymore. You should be adding these to:

fern/docs/pages/manual/llms.mdx

instead.

Reply:
done